src/annotations/standard/ParamAnnotation.php

```diff
@@ -53,7 +53,10 @@ public static function parseAnnotation($value)
     {
         $parts = explode(' ', trim($value), 3);
 
-        return array('type' => $parts[0], 'name' => substr($parts[1], 1));
+        if (isset($parts[1]))
+            return array('type' => $parts[0], 'name' => substr($parts[1], 1));
+        else
+            return array('type' => 'unknown', 'name' => substr($parts[0], 1));
```

Comment:
Having the type in name rather than the name in type seems less problematic to me.

Could try and identify primitives like int, string and classes by checking for a /, but otherwise...